index.html

```diff
@@ -1,136 +1,164 @@
-          </ul>
-          <button class="main-nav__toggle main-nav__toggle--close"></button>
-          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
-        </nav>
-
-        <div calss="intro">
-          <ul class="intro-list">
-            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
-            <li class="intro-list__item intro-list__item--middle">Sedona</li>
-            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
-          </ul>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="description">
+  <title>Sedona cosik</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Sans:400,700">
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <!-- main page -->
+    <header class="page-header">
+      <nav class="main-nav">
+        <ul class="main-nav__list">
+          <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+          <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+          <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+          <li class="main-nav__item"><a href="#">HTML Academy</a></li>
+        </ul>
+        <div class="main-nav__toggle main-nav__toggle--close">
+          <div class="main-nav__menu-bar main-nav__menu-bar--top"></div>
+          <div class="main-nav__menu-bar main-nav__menu-bar--middle"></div>
+          <div class="main-nav__menu-bar main-nav__menu-bar--bottom"></div>
+        </div>
+        <a class="main-nav__link" href="index.html"><img class="main-nav__logo" src="images/logo.png"  alt="Sedona logo"></a>
+      </nav>
+
+      <div class="intro">
+        <div class="intro-images">
+          <h3 class="intro-images__header">Welcome to gorgeous Sedona. "Because the Grand Canyon sucks!"<h3>
+          </div>
         </div>
+        <div class="header-separator"></div>
       </header>
 
+      <!-- <section class="header-separator"> -->
+        <!-- <svg class="header-separator__svg" xmlns="http://www.w3.org/2000/svg" -->
+              <!-- version="1.1" width="100%" height="50" -->
+              <!-- viewBox="0 0 100 10" preserveAspectRatio="none"> -->
+            <!-- <polygon points="0 2, 50 10, 100 2, 100 500, 0 500"></polygon> -->
+        <!-- </svg> -->
+      <!-- </section> -->
+
       <main class="main-content">
         <section class="preamble">
           <div class="preamble__inner">
-            <h2 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h2>
+            <h3 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h3>
             <p class="preamble__paragraph">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
           </div>
+          <div class="preamble-separator"></div>
```

Comment:
what's the difference between header-separator and preamble-separator? If those are two separate blocks, then they shouldn't probably if they duplicate code of each other. Use one. Another option is to make it a paren block feature(preamble, header) then it's fine to have duplication.